cmd/main.go

```diff
@@ -60,6 +62,23 @@ func main() {
 		log.Fatalf("Could not load config: %v", err)
 	}
 
+	// Load sub-attesters from the path specified in config.yaml
+	pluginLoader, err := plugin.CreateGoPluginLoader(
+		cfg.PluginDir, log.Named("plugin"))
+
+	if err != nil {
+		log.Fatalf("could not create plugin loader: %v", err)
+	}
+
+	pluginManager, err := plugin.CreateGoPluginManagerWithLoader(
+		pluginLoader, log.Named("builtin"))
+
```

Comment:
Can you replace the calls to `plugin.CreateGoPluginLoader()` & `plugin.CreateGoPluginManagerWithLoader()` with a single call to `plugin.CreateGoPluginManager()` ?

Reply:
Agree, there's currently no `builtin` loader here in ratsd as opposed to `veraison/services`. We might want to add a `secure loader` in ratsd as the option in the future, and we'll have to separate the loader and the manager again then.